logging/google/cloud/logging/logger.py

```diff
@@ -379,13 +410,17 @@ def log_text(self, text, labels=None, insert_id=None, severity=None,
 
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (optional) timestamp of event being logged.
+
+        :type resource :class:``google.cloud.logging.resource`
+        :param resource: (Optional) Monitored resource of the entry
         """
         self.entries.append(
             ('text', text, labels, insert_id, severity, http_request,
-             timestamp))
+             timestamp, resource))
```

Comment:
Will leave another comment, but basically each entry has a resource and a batch a resource that is used for any entries that don't have a resource. self.resource is batch resource, here resource is the entry resource.